test/unittests/t_util.cc

```diff
@@ -792,7 +792,10 @@ TEST_F(T_Util, RfcTimestamp) {
   time_t time1 = time(NULL);
   string str = RfcTimestamp();
   strptime(str.c_str(), format, &tm);
-  time_t time2 = mktime(&tm);
+  time_t time2 = mktime(&tm) - timezone;
+  if (tm.tm_isdst > 0) {
```

Comment:
I had no issues with this while testing it, but just in case if it is in daylight saving time (i.e., during summer) it will subtract one hour